package.json

```diff
@@ -67,7 +68,9 @@
         "@semantic-release/github": "^8.0.5",
         "@semantic-release/npm": "^9.0.1",
         "@semantic-release/release-notes-generator": "^10.0.3",
+        "adapt-cli": "^3.0.7",
```

Comment:
No sure this is needed as `npm install -g adapt-cli` the module is installed globally